src/main/java/com/mappy/fpm/batches/tomtom/helpers/RoadTagger.java

```diff
@@ -122,6 +122,9 @@ public RoadTagger(SpeedProfiles speedProfiles, GeocodeProvider geocodeProvider,
 
         tagRoute(feature, tags, id, isOneway, isReversed);
 
+        addTagIf("ramp_type", () -> feature.getInteger("RAMP").equals(1) ? "exit" : "entrance", 
```

Comment:
I see that we are potentially evaluating 4 times `feature.getInteger("RAMP")` would it be possible to compute it only once?
```java
Integer ramp = feature.getInteger("RAMP");
addTagIf("ramp_type", () -> ramp.equals(1) ? "exit" : "entrance", 
            ofNullable(ramp).isPresent() && (ramp.equals(1) || ramp.equals(2)), tags);
```

Reply:
remarque prise en compte